src/onboarding2/Templates/TemplateCard.js

```diff
@@ -0,0 +1,78 @@
+import React, { useCallback } from 'react'
+import { Card, textStyle, GU, Button, useTheme } from '@aragon/ui'
+import { OrgTemplateType } from '../../prop-types'
+
+function TemplateCard({ onOpen, template }) {
+  const theme = useTheme()
+  const handleDetailsClick = useCallback(() => {
+    onOpen(template.id)
+  }, [onOpen, template.id])
+  return (
+    <Card
```

Comment:
cc @dizzypaty Even though we have an explicit button, it might be nice to have the entire card clickable (or only on small screens)